go/tasks/pluginmachinery/catalog/hashing_test.go

```diff
@@ -593,3 +593,33 @@ func TestNoInputValues(t *testing.T) {
 	assert.Equal(t, "GKw-c0PwFokMUQ6T-TUmEWnZ4_VlQ2Qpgw-vCTT0-OQ", hashDupe)
 	assert.Equal(t, hashDupe, hash)
 }
+
+// Ensure that hash values on maps and collections are carried over
+// into representing literals.
+func TestHashCarryOnContainers(t *testing.T) {
```

Comment:
can you move these tests to the table-tests in `TestHashLiteralMap_LiteralsWithHashSet` instead?

Reply:
Done in f4d830ee7b44b30399082eaaadb069e740fd2ebb.